README.md

```diff
@@ -1,5 +1,5 @@
 ## Welcome to KeyCryptoVC Base
   		  
  - [Projects](projects/projects.md)  

```

Comment:
Here you link to `projects/projects.md` but you keep putting changes into `project.md`